pandas/core/generic.py

```diff
@@ -5716,24 +5716,81 @@ def clip_upper(self, threshold, axis=None, inplace=False):
 
     def clip_lower(self, threshold, axis=None, inplace=False):
         """
-        Return copy of the input with values below given value(s) truncated.
+        Return copy of the input with values below given value(s) trimmed.
+
+        If an element value is below the threshold, the threshold value is
+        returned instead.
 
         Parameters
         ----------
         threshold : float or array_like
-        axis : int or string axis name, optional
+            Lower value(s) to which the input value(s) will be trimmed.
+        axis : {0 or 'index', 1 or 'columns'}
             Align object with threshold along the given axis.
         inplace : boolean, default False
             Whether to perform the operation in place on the data
-                .. versionadded:: 0.21.0
+                .. versionadded:: 0.21.0.
```

Comment:
No period after versionadded. Also, does this render? I think it needs to be down one more line and aligned with text directly above

Reply:
It rendered OK, but for consistency with other docstrings I am adding the extra line and shifting it to the left.

Re the period, the validation script complains if it's not there. See discussion in https://github.com/pandas-dev/pandas/pull/20227.

Reply:
Hmm OK. @datapythonista @jorisvandenbossche another issue of where I think the script is incorrectly calling out a missing period - we shouldn't be adding this to the end of a `versionadded` directive right?

Reply:
Yes, the script is wrong when versionadded is present. Please ignore the error in the validation.